src/storage/v3/vertex_accessor.hpp

```diff
@@ -94,7 +94,10 @@ class VertexAccessor final {
 
   Result<size_t> OutDegree(View view) const;
 
-  Gid Gid() const noexcept { return vertex_->gid; }
+  Gid Gid() const noexcept {
+    // TODO(antaljanosbenjamin): remove this whole function.
```

Comment:
Is this going to be part of this PR?

Reply:
No, this can only be removed when we have schema.